libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -29,7 +29,7 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
     // @Param: _TYPE
     // @DisplayName: Rangefinder type
     // @Description: What type of rangefinder device that is connected
-    // @Values: 0:None,1:Analog,2:APM2-MaxbotixI2C,3:APM2-PulsedLightI2C,4:PX4-I2C,5:PX4-PWM,6:BBB-PRU,7:LightWareI2C,8:LightWareSerial
+    // @Values: 0:None,1:Analog,2:APM2-MaxbotixI2C,3:APM2-PulsedLightI2C,4:PX4-I2C,4:TeraRangerOne-I2C,5:PX4-PWM,6:BBB-PRU,7:LightWareI2C,8:LightWareSerial
```

Comment:
as far as im aware yes, the reason for the separation is so other paramaters like the min and max can be predefined.
although randy did bring up a  good point about gcs's possible wont know what to display if there are 2, 4's